src/CoreWCF.Primitives/src/CoreWCF/XmlSerializerFormatAttribute.cs

```diff
@@ -39,7 +39,7 @@ public OperationFormatUse Use
             }
         }
 
-        internal bool IsEncoded
+        public bool IsEncoded
         {
```

Comment:
Leave as internal and instead replace call site to use XmlSerializerFormatAttribute.Use == OperationFormatUse.Encoded

Reply:
This is addressed in c3c91c1aadd3e87bec21844f548f79ad8504f207.